install/install.go

```diff
@@ -241,6 +241,16 @@ var operatorClusterRole = &rbacv1.ClusterRole{
 }
 
 func (k *K8sInstaller) generateAgentDaemonSet() *appsv1.DaemonSet {
+	healthCheckPort := 9876
+
+	version, err := utils.ParseCiliumVersion(k.params.Version, k.params.BaseVersion)
+	if err != nil {
+		k.Log("Unable to parse the provided version %q, assuming %v for health checking port compatibility", k.params.Version, defaults.Version)
+	} else if version.GE(versioncheck.MustVersion("v1.10.12")) ||
+		version.GE(versioncheck.MustVersion("v1.11.6")) {
```

Comment:
i think v0.10 CLIs are only meant to support cilium v1.10 installation, so i think we can just do this instead:

```suggestion
	} else if version.GE(versioncheck.MustVersion("v1.10.12")) {
```

Reply:
Agree, I don't think installation for Cilium v1.11+ will work with cilium-cli v0.10.x.